lib/posix-process/Config.uk

```diff
@@ -4,50 +4,44 @@ menuconfig LIBPOSIX_PROCESS
 
 if LIBPOSIX_PROCESS
 
-menuconfig LIBPOSIX_PROCESS_PIDS
-	bool "Process and thread IDs"
+menuconfig LIBPOSIX_PROCESS_MULTITHREADING
+	bool "Multithreading support"
 	select LIBUKSCHED
+	select LIBSYSCALL_SHIM
```

Comment:
Why do we need syscall shim as mandatory now?

Reply:
Okay so these are macros that are always compiled in right. Will update, thanks 👍🏼